scripts/service/driver/start

```diff
@@ -11,13 +11,15 @@ then
 fi
 
 export HOME=/run/sysflow
-if [ "${DRIVER_TYPE}" == "ebpf" ] || [ -z "${DRIVER_TYPE}" ]; then
+if [ "${DRIVER_TYPE}" == "ebpf-core" ] || [ -z "${DRIVER_TYPE}" ]; then
+	/etc/sysflow/driver/falcoctl driver config --type modern_ebpf
+elif [ "${DRIVER_TYPE}" == "ebpf" ]; then
     export DRIVER_TYPE="ebpf"
     if [ -z "${FALCO_BPF_PROBE}" ]; then
        export FALCO_BPF_PROBE=""
```

Comment:
Isn't `FALCO_BPF_PROBE` deprecated in Falco 0.38? I think we need to clean this up here, and also in the [systemd service file](https://github.com/sysflow-telemetry/sf-collector/blob/a678da64606294789a58bf449a0b7ee688117489/scripts/service/systemd/sysflow-collector.service#L11), where we select which environment variables to expose to the service. 

I noticed a new environment variable in the docker-run.sh for the examples. Does it need to be passed here?